### DISCUSSION TITLE
Is there an example site published with ObsidianPublisher I could take a look at?

### DISCUSSION THREAD

**Discussion by tsouchlarakis** [ObsidianPublisher/obsidian-github-publisher#181] [General]
I'm exploring this repo, and wonder whether there's a sample site I can view before assessing whether this will work for my use case.

**Comment by Mara-Li** [comment]
You could check mine:
https://www.mara-li.fr/

Repo @ : lisandra-dev/mara-li

**Reply by tsouchlarakis** [reply]
Very nice, thank you!
